indico/modules/events/editing/operations.py

```diff
@@ -205,6 +211,10 @@ def create_revision_comment(revision, user, text, internal=False):
     comment = EditingRevisionComment(user=user, text=text, internal=internal)
     revision.comments.append(comment)
     db.session.flush()
+    editable = revision.editable
+    if editable.can_perform_submitter_actions(user):
+        notify_submitter_comment(revision)
```

Comment:
Pass the `user` here as well instead of relying on `revision.submitter` in there (there can be multiple people with submitter permissions and the email should mention the user who actually posted the comment).

Alternatively, pass the `comment` to the function since that one already has both the `user` and `revision`...